metacatalog/ext/standards_export/extension.py

```diff
@@ -78,6 +81,17 @@ def wrapper_api(session: Session, id_or_uuid: Union[int, str], config_dict: dict
         api.catalog.create_iso19115_xml = wrapper_api
 
 
+    @classmethod
+    def init_cli(cls, subparsers, defaults):
+        myparser = subparsers.add_parser('standards-export', parents=[defaults], help="Export metadata in standard format as .xml files.")
+        myparser.add_argument('--format', choices=['iso19115'], type=str, required=True, help="Metadata standard format.")
+        myparser.add_argument('--uuid', type=str, help="uuid of the entry to export, must be specified if --id or --all is not specified.")
+        myparser.add_argument('--id', type=int, help="id of the entry to export, must be specified if --uuid or --all is not specified.")
```

Comment:
we could also think of accepting a list of entry ids or uuids and only if that is empty, which means export all. Then we could also skit the --all flag.

If that is too complicated to implement, I am also fine with this approach.

Reply:
I have thought of that as well, and I can implement it that way, especially that multiple ids / uuids are accepted.  
I thought it would be a good idea to force the user to explicitly set the `--all` flag to prevent the creation of a large amount of XML files by accident.

Reply:
@mmaelicke what do you think, should I drop the `--all` flag and export all entries by default if no id or uuid is given?  
I think it is better to force the user to specify what he wants to export and print error messages if neither `--id`, `--uuid` nor `--all` is set.

Reply:
Maybe drop `--id` and `--uuid`, but rather use a positional argument. So the user **has to** pass `[entries]`, which can be an id or uuid (you can tell by the type, as ids are int and uuid are str). Then the cmd function can check if the `len(entries= == 0`  and only accept that if `--all` is also set, which defaults to False. If `--all`  is not given, we can print a warning. What do you think?

Reply:
I just implemented the command with the positional argument `entries`, you can have a look at it and see if that is fine already.